core/modules/qos_measure.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright(c) 2021 Open Networking Foundation
+ */
+
+#ifndef BESS_MODULES_QOS_MEASURE_H_
+#define BESS_MODULES_QOS_MEASURE_H_
+
+#include <unordered_map>
```

Comment:
STL hashmaps are generally trash, let's see if we can use something better.

Reply:
stl hashmap are easy to use but not thread safe (multiple thread writing could produce crash/corruption). 

we shifted to dpdk rte_hash as its support multithreading efficiently where as BESS cuckoo hash does'nt.